lib/importers/daily_calls/tp/call_record.rb

```diff
@@ -0,0 +1,30 @@
+module Importers
+  module DailyCalls
+    module TP
+      class CallRecord
+        MINIMUM_CALL_TIME = 10
+
+        def initialize(row)
+          @row = row
```

Comment:
as discussed - extract outcome method as that gives reference to each @row[..] in the class.